ecl/hql/hqlgram.y

```diff
@@ -1742,6 +1743,11 @@ persistOpt
     : fewMany
     | expireAttr
     | clusterAttr
+    | REFRESH '(' expression ')'
```

Comment:
@ghalliday I ironically missed this the very first time and didn't question your note after until your much later comment regarding additional syntax checks - might this be clearer/cleaner if constExpression, even though the normalize  below does the same? I had even wondered if it might be cleaner to use optContBoolArg, since MULTIPLE below also allows for the opt ( expr ) but it doesn't make sense since it would (make sense semantically to) default to true which it does when not present anyhow.

However, the real question that lead me to look at this section again, why is it constant at all? The checks are in both eclagent and roxie themselves, but perhaps this sheds light on my previous comment regarding the evaluation and snippet from the langref, i.e. if it wasn't constant then it couldn't 'hoist' and eval "...outside of any given evaluation order"? REFRESH(false) only re-builds if the persist doesn't exist, so it doesn't do so if the ecl has changed and thus if the record formats have changed (as @jakesmith noted), I'm starting to wonder if this is attempting to solve this issue but I don't follow how it does/would? Also, since the ecl isn't crc'd against, with refresh(false), it seems that a good semantic use (or at least one easily implied/assumed) is to have this as non-constant - almost feels like its primary functional use? 
Perhaps I missed a previous conversation regarding all this as by design? Has it been made explicit to David (I can't find anywhere) that this option is constant? Which reminds me, Gavin mentioned before needing a test to test the rec format change handling, do we still want one?